docs-en/chapter_stack_and_queue/queue.md

```diff
@@ -1,14 +1,14 @@
 # Queue
 
-"Queue" is a linear data structure that follows the First-In-First-Out (FIFO) rule. As the name suggests, a queue simulates the phenomenon of lining up, where newcomers join the back of the queue, and people at the front of the queue leave one by one.
+"Queue" is a linear data structure that follows the First-In-First-Out (FIFO) rule. As the name suggests, a queue simulates the phenomenon of lining up, where newcomers join at the rear of the queue, and people at the front leave one by one.
 
-As shown in the figure below, we call the front of the queue the "head" and the back the "tail." The operation of adding elements to the tail of the queue is termed "enqueue," and the operation of removing elements from the head is termed "dequeue."
+As shown in the figure below, we call the front of the queue the "head" and the back the "tail." The operation of adding elements to the rear of the queue is termed "enqueue," and the operation of removing elements from the front is termed "dequeue."
 
 ![Queue's First-In-First-Out Rule](queue.assets/queue_operations.png)
 
 ## Common Operations on Queue
 
-The common operations on a queue are shown in the table below. Note that method names may vary across different programming languages. Here, we adopt the same naming convention as used for stacks.
+The common operations on a queue are shown in the table below. Note that method names may vary across different programming languages. Here, we use the same naming convention as that for stacks.
 
 <p align="center"> Table <id> &nbsp; Efficiency of Queue Operations </p>
 
```

Comment:
In the sample code section, "# Access the front element" can be ambiguous, probably use "the first element".